### PR TITLE
AM Updates 4.9

### PR DESCRIPTION
**Pull request by amorning3**

### CODE REVIEW

File: 101-workshop-nodejs/ex4-exposing-apis-from-actions/concepts.md [hunk 1]
```diff
@@ -19,30 +19,27 @@
 
 # Concepts
 
-## Web Actions
+## Web actions
 
-Cloud Function actions can be annotated with a special flag (i.e., `--web true`) at creation to convert them into "web actions". The result is the corresponding creation of a public URL that can be used to trigger the action from any web app.
+Cloud Function actions can be annotated with a special flag, such as`--web true`, at creation to convert them into [web actions](https://cloud.ibm.com/docs/openwhisk?topic=cloud-functions-actions_web). The result is the corresponding creation of a public URL that can be used to trigger the action from any web app.
```
Comment (mrutkows) [inline 101-workshop-nodejs/ex4-exposing-apis-from-actions/concepts.md:24]: It is not "such as" it "is" (only) `--web true`

File: 101-workshop-nodejs/ex4-exposing-apis-from-actions/concepts.md [hunk 2]
```diff
@@ -19,30 +19,27 @@
...
 
-Web actions can then also be invoked via HTTP requests without user authentication where the HTTP request parameters are automatically converted in event parameters. Web actions are able to control the HTTP response headers and body to support any content-types, manage cookies, perform HTTP redirects directly.
+Web actions can then be invoked via HTTP requests without user authentication where the HTTP request parameters are automatically converted in event parameters. Web actions are able to control the HTTP response headers and body to support any content types, manage cookies, and perform HTTP redirects directly.
 
-In addition, requesters can specify the `content-type` they want for the response as one of `.json`, `.html`, `.http`, `.svg` or `.text`. For convenience, the `.http` extension is the default.
```
Comment (mrutkows) [inline 101-workshop-nodejs/ex4-exposing-apis-from-actions/concepts.md:28]: There is a limited (fixed set) of supported content types so it is "one of" that set

File: 101-workshop-nodejs/ex4-exposing-apis-from-actions/concepts.md (hunk 1 above)
Comment (amorning3) [reply 101-workshop-nodejs/ex4-exposing-apis-from-actions/concepts.md:24]: Then can it just read like: "Cloud Function actions can be annotated with a special flag,`--web true`, at creation to.."

File: 101-workshop-nodejs/ex4-exposing-apis-from-actions/concepts.md (hunk 2 above)
Comment (amorning3) [reply 101-workshop-nodejs/ex4-exposing-apis-from-actions/concepts.md:28]: It just isn't grammatically correct. Can it be "as one of the following: `.json`..."?

File: 101-workshop-nodejs/ex4-exposing-apis-from-actions/concepts.md (hunk 1 above)
Comment (mrutkows) [reply 101-workshop-nodejs/ex4-exposing-apis-from-actions/concepts.md:24]: works nicely

File: 101-workshop-nodejs/ex4-exposing-apis-from-actions/concepts.md (hunk 2 above)
Comment (mrutkows) [reply 101-workshop-nodejs/ex4-exposing-apis-from-actions/concepts.md:28]: sure

**Pull request merged**